### ISSUE TITLE
tracking google drive documents, sheets, and presentations

### ISSUE THREAD

**Issue by jhpoelen** [bio-guoda/preston#284]
Many folks use Google Drive to keep, collaborate on, and share documents/spreadsheets/presentations.

However, keeping track of these google docs/sheets/slides outside of Google Drive requires clicking on export buttons.

Suggest to implement a preston tracker feature that allows for tracking of google drive resources, and exporting them is suitable formats.

For instance, given a google doc at https://docs.google.com/document/d/1LMnC0lUw_DGIQV5Pa4lZhe_7-SIR-otgHSHDNkAwD7Q/edit , I'd like to be able to say:

```
preston track "https://docs.google.com/document/d/1LMnC0lUw_DGIQV5Pa4lZhe_7-SIR-otgHSHDNkAwD7Q/edit"
```

and preston would take the current version of the documents and export it in available formats.

**Comment by jhpoelen** [comment]
A first pass at Google Drive resource tracking is now available via - 

```
preston track [google drive resource url]
```

for instance,

```
preston track "https://docs.google.com/document/d/1LMnC0lUw_DGIQV5Pa4lZhe_7-SIR-otgHSHDNkAwD7Q/edit"
```

produced attached data.zip with provenance hash hash://sha256/8d06be909b45d5cc906d879eb9c5a21bbfd7e89904148eb1b630384cea39a84c .

[data.zip](https://github.com/bio-guoda/preston/files/15242338/data.zip)

for extracting specific resources, you can say things like:

to produce attached doc.pdf . 

[doc.pdf](https://github.com/bio-guoda/preston/files/15242347/doc.pdf)


fyi @zedomel @filipi-soares . . . ready to track all the google drive documents / sheets you'd like to keep and cite?

**Comment by jhpoelen** [comment]
google sheets / doc / presentation tracking is now available via [Preston v0.5.8](https://github.com/bio-guoda/preston/releases/tag/0.8.5) .

fyi @daniel-mietchen - I wonder how we can tie the many google docs to wikidata entries in a secure and verifiable way . 

Perhaps a citation with provenance provided by Preston history.
